ipt_connect/IPTdev/parameters.py

```diff
@@ -2,7 +2,10 @@
 
 # Views parameters
 app_version = "dev"     # keyword for url parsing
-NAME = "IPT dev"
+NAME = {
```

Comment:
hum...names should be generic in IPTdev :)

Reply:
Fixed: https://github.com/nickkolok/ipt_connect/commit/211b3682af1dfb20eb78daaa02a34ffaf8ec27e6